Packs/Lastline/Integrations/Lastline_v2/Lastline_v2_test.py

```diff
@@ -69,3 +70,16 @@ def test_get_report_context(path):
             json_obj = json_obj.read()
             out_dict = json.loads(json_obj)
         get_report_context(out_dict)
+
+
+def raise_exception():
+    raise DemistoException('error Missing required field \'uuid\'.')
+
+
+def test_credentials_not_part_of_params(mocker):
+    mocker.patch.object(demisto, 'params', return_value={'url': 'testurl.com',
+                                                         'api_key': 'apikey',
+                                                         'api_token': 'apitoken'})
+    mocker.patch.object(demisto, 'command', return_value='test-module')
+    mocker.patch.object(Client, 'get_report', side_effect=raise_exception)
```

Comment:
This is not a good way to validate the exception, need to use `with raises` instead. In general, I don't understand what are we testing here. Ping me when you are getting back to this.

Reply:
explained to you the test, in our zoom conversation